tests/e2e/testdata/scenarios/fields/gkehubfeaturemembership/set_unset/script.yaml

```diff
@@ -0,0 +1,143 @@
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+WRITE-KUBE-OBJECT: false
+apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
+kind: Service
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id: mock-project
+    cnrm.cloud.google.com/disable-dependent-services: "false"
+  name: container.googleapis.com
+---
+WRITE-KUBE-OBJECT: false
+apiVersion: container.cnrm.cloud.google.com/v1beta1
+kind: ContainerCluster
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id: mock-project
+  name: containercluster-${uniqueId}
+spec:
+  location: us-central1-a
+  workloadIdentityConfig:
+    workloadPool:  mock-project.svc.id.goog
+---
+WRITE-KUBE-OBJECT: false
+apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
+kind: Service
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id:  mock-project
+    # Abandon the resource to unblock cleanup; Otherwise it fails with error message
+    # "Error waiting for api to disable ... ensure there are no more resources managed
+    # by this service." Likely there is some delay in GKEHub resources clean up.
+    cnrm.cloud.google.com/deletion-policy: "abandon"
+  name: gkehub.googleapis.com
+---
+WRITE-KUBE-OBJECT: false
+apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
+kind: Service
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id:  mock-project
+    cnrm.cloud.google.com/disable-dependent-services: "false"
+  name: anthosconfigmanagement.googleapis.com
+---
+WRITE-KUBE-OBJECT: false
+apiVersion: gkehub.cnrm.cloud.google.com/v1beta1
+kind: GKEHubFeature
+metadata:
+  name: gkehubfeature-${uniqueId}
+spec:
+  projectRef:
+    external: projects/mock-project
+  location: global
+  resourceID: configmanagement
+---
+WRITE-KUBE-OBJECT: false
+apiVersion: gkehub.cnrm.cloud.google.com/v1beta1
+kind: GKEHubMembership
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id: mock-project
+  name: gkehubmembership-${uniqueId}
+spec:
+  location: global
+  authority:
+    issuer: https://container.googleapis.com/v1/projects/mock-project/locations/us-central1-a/clusters/containercluster-${uniqueId}
+  description: A sample GKE Hub membership
+  endpoint:
+    gkeCluster:
+      resourceRef:
+        name: containercluster-${uniqueId}
+---
+apiVersion: gkehub.cnrm.cloud.google.com/v1beta1
+kind: GKEHubFeatureMembership
+metadata:
+  name: gkehubfeaturemembership-${uniqueId}
+spec:
+  projectRef:
+    external: projects/mock-project
+  location: global
+  membershipRef:
+    name: gkehubmembership-${uniqueId}
+  featureRef:
+    name: gkehubfeature-${uniqueId}
+  configmanagement:
+    binauthz:
```

Comment:
Why do we test binauthz here?

Reply:
binauthz is deprecated already. I don't think we need to have coverage for it in the test.